recipes/mold/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "1.3.1":
-    folder: all
+    folder: 1.3.1
+  "1.4.2":
```

Comment:
This got outdated since you included version 1.5. Would it make sense to rename the folder to `all`? So we have an `all` folder for current recipes and a legacy `1.3.1` folder. It looks like mold changes minor versions often since 1.4 is relatively new.

Reply:
Yes, makes sense